DeepSpeech.ipynb

```diff
@@ -1386,7 +1428,7 @@
     "            'total_batches_validation': data_sets.dev.total_batches, \\\n",
     "            'total_batches_test': data_sets.test.total_batches, \\\n",
     "            'data_set': { \\\n",
-    "                'name': ds_importer\n",
+    "                'name': ds_importer \\\n",
```

Comment:
Aesthetics only: For the (added) backslash not being visually glued to ```ds_importer``` like ```ds_importer\```. I prefer ```ds_importer \```.